python_bindings/tutorial/lesson_02_input_image.py

```diff
@@ -18,7 +18,7 @@ def main():
     # brightens an image.
 
     # First we'll load the input image we wish to brighten.
-    image_path = os.path.join(os.path.dirname(__file__), "../../tutorial/images/rgb.png")
+    image_path = os.path.join(os.environ["TEST_IMAGES_DIR"], "rgb.png")
```

Comment:
The intended usage for these tutorials is someone typing "python lesson_02_input_image.py" at the shell. Won't this break that?